docs/topics/web-require-auth.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/web-console-guide/master.adoc
 
+:_content-type: REFERENCE
```

Comment:
@stoobie @sbeskin-redhat This is a procedure. The file needs ".Procedure" added to it in the correct place.

Reply:
@RichardHoch  Yes, you are right.